cpp/modules/deck.gl/core/src/lib/layer-context.h

```diff
@@ -21,6 +21,8 @@
 #ifndef DECKGL_CORE_LAYER_CONTEXT_H
 #define DECKGL_CORE_LAYER_CONTEXT_H
 
+#include <dawn/webgpu_cpp.h>
```

Comment:
Nit: Would it make sense to only include dawn headers inside luma.gl headers, and include luma.gl/core or luma.gl/webgpu here instead?